src/components/node/Node.jsx

```diff
@@ -98,9 +100,18 @@ export default class Node extends React.Component {
         let label;
         let node;
 
+        //Determine if we have a rectangle or a square size
+        var sizeWidth = size;
+        var sizeHeight = size;
+
+        if (this.props.width != undefined && this.props.height != undefined) {
```

Comment:
Another good practice in JS is to always use `===` instead of `==`, so you could consider replacing `!=` with `!==`. 

Also, you probably want to extend this condition to all of the non-falsy values instead of comparing only to `undefined`. To do that, you could shorten this condition to `(this.props.width && this.props.height)` to check for `null`, `false`, `NaN`, `undefined` or empty string all at once. This condition will return `true` only when both `width` and `height` have an actual value.  

Another suggestion would be to reorganize this piece of code to assign `sizeWidth` and `sizeHeight` in a single line, so we can avoid this `if` block over here. What about something like this? 

```
const useRectangle = this.props.width && this.props.height;
const sizeWidth = useRectangle ? this.props.width: size;
const sizeHeight = useRectangle ? this.props.height : size;
```

Reply:
This should fail in the `lint` check, otherwise, I will add [eqeqeq](https://eslint.org/docs/rules/eqeqeq#always) rule to our lint config.

Reply:
This is what is there now. It passes lint.
```
    //Determine if we have a rectangle or a square size
        const useRectangle = this.props.width && this.props.height;
        const sizeWidth = useRectangle ? this.props.width : size;
        const sizeHeight = useRectangle ? this.props.height : size;
```

Reply:
Sorry if I was not clear, I meant the `this.props.width != undefined` the usage of `!=` instead of `!==`

Reply:
@danielcaldas, what you said made sense. Based on @LonelyPrincess's comments that line is not in the code anymore.